### DISCUSSION TITLE
Noob here - how to see individual clients in ADH

### DISCUSSION THREAD

**Discussion by Aloe-recite** [trinib/Adguard-Wireguard-Unbound-Cloudflare#74] [Help]
Hi there!
I managed to set up this great combo, not without some hiccups but I'm a total noob so that was to be expected :)
I see that ADH show only one connected client, which is the Wireguard interface I think. My issue is that therefore I can't seem to be able to distinguish exactly WHICH wireguard peer (i.e. which device) is doing what, and I can't use ADH personalisation for each device. I think this is to be expected, but is there any way to monito individual devices / wireguard peers in ADH?
Please be gentle, I'm a total noob so I might be doing something wrong or not fully understanding things :)
Thank you!

**Comment by trinib** [comment]
Hey if you have not figured it out already , yes you can rename your client under settings / client settings. I think when connected to wireguard it shows something like 10.7.0.3 which is its local address.  Just see which device is that address an rename it in AGH.
